tests/meltano/cli/test_install.py

```diff
@@ -242,3 +245,21 @@ def test_clean_install(
 
             mappers = [m for m in commands[0][1] if m == mapper]
             assert len(mappers) == 3
+
+
+# un_engine_uri forces us to create a new project, we must do this before the
+# project fixture creates the project see
+# https://github.com/meltano/meltano/pull/6407#issuecomment-1200516464
+# For more details
+def test_new_folder_should_autocreate_on_install(
+    un_engine_uri, project_function, cli_runner
```

Comment:
Tried using `project` instead of `project_function` no luck. I spent more time than I probably should have on this so I think this working is probably good enough. We could make a follow up issue regarding https://github.com/meltano/meltano/pull/6407#issuecomment-1200516464 

wdyt @WillDaSilva  ?